com.unity.ml-agents/Runtime/Actuators/ActionSpecs.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.MLAgents.Policies;
+
+namespace Unity.MLAgents.Actuators
+{
+    /// <summary>
+    /// Defines the structure of an Action Space to be used by the Actuator system.
+    /// </summary>
+    internal readonly struct ActionSpecs
```

Comment:
Maybe `ActionSpec`? I think on the python side, we use `BehaviorSpec`

Reply:
I'm ok with either. @vincentpierre what do you say?

Reply:
Spec it is